json/src/main/scala/com/gu/contentapi/json/CirceDecoders.scala

```diff
@@ -237,4 +237,5 @@ object CirceDecoders {
   implicit val entitiesResponseDecoder: Decoder[EntitiesResponse] = deriveDecoder
   implicit val pillarsResponseDecoder: Decoder[PillarsResponse] = deriveDecoder
   implicit val embedReachDecoder: Decoder[EmbedReach] = deriveDecoder
+  implicit val linkElementFieldsDecoder: Decoder[LinkElementFields] = deriveDecoder
```

Comment:
I'm slightly out of my depth here but are these changes to `CirceEncoders` and `CirceDecoders` needed? I don't recall changing anything here for the List element work

Reply:
We added these changes to fix a test failure. You can see the results of the test GH action for the initial commit [here](https://github.com/guardian/content-api-models/actions/runs/15322041445/job/43107770626), and the test was failing for the block level decoder and encoder without adding this in.

Reply:
I see, sounds reasonable 👍